samples/speech/index.html

```diff
@@ -71,19 +73,23 @@
             
 
             // Note that chrome automatically blocks speech if the html file is loaded from disk. You can run a server locally 
-            // or launch chrome with the following option:
+            // or launch chrome (close all the existing chrome browsers) with the following option:
             // chrome.exe --allow-file-access-from-files <sampleHtmlFile>
-            var speechOptions = {
-                speechRecognizer: new BotChat.SpeechRecognition.BrowserSpeechRecognizer(),
-                speechSynthesizer: new BotChat.SpeechSynthesis.BrowserSpeechSynthesizer()
-            }
+            // var speechOptions = {
+            //     speechRecognizer: new BotChat.Speech.BrowserSpeechRecognizer(),
+            //     speechSynthesizer: new BotChat.Speech.BrowserSpeechSynthesizer()
+            // }
 
             // // Option 3 : Cognitive services speech recognition using API key (cross browser, speech priming support)
             //
-            // var speechOptions = {
-            //     speechRecognizer: new BotChat.SpeechRecognition.CognitiveServicesSpeechRecognizer( { subscriptionKey: 'YOUR_COGNITIVE_SPEECH_API_KEY' } ),
-            //     speechSynthesizer: new BotChat.SpeechSynthesis.BrowserSpeechSynthesizer()
-            // }
+            var speechOptions = {
+                speechRecognizer: new CognitiveServices.SpeechRecognizer( { subscriptionKey: 'YOUR_COGNITIVE_SPEECH_API_KEY' } ),
```

Comment:
Is the requirement here that they update the JavaScript and add in their key? This seems like it should be passed in as a constructor parameter.